### PR TITLE
Add Vantac Lite TX and RX Targets

### PR DESCRIPTION
**Pull request by pkendall64**

### CODE REVIEW

**Review by CapnBry — APPROVED** [review]
TX looks good to me, flashes to my sample unit and runs. RGB is correct orientation, wifi has the standard range, usb serial for logging works as well as firmware upload. The TX has a button that I assume is in GPIO0 if we want to add that line, but it is inaccessible with the cover on so not sure if we want to add that.
10mw = 13mW
25mW = 29mW
50mW = 52mW
100mW = 114mW
250mW = 269mW
500mW = 320mW

Did we get RXes for testing from them? I don't see one in my testing pile, but things look so much the same I might have mixed it in with something else.

File: src/include/target/Vantac_2400_RX.h [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#define GPIO_PIN_NSS            15
+#define GPIO_PIN_BUSY           5
+#define GPIO_PIN_DIO1           4
+#define GPIO_PIN_MOSI           13
+#define GPIO_PIN_MISO           12
+#define GPIO_PIN_SCK            14
+#define GPIO_PIN_RST            2
+#define GPIO_PIN_LED            16
+#define GPIO_PIN_BUTTON         0
+#define GPIO_PIN_RX_ENABLE      9 //enable pa
```
[the remark this answers is not in the document]
Comment (pkendall64) [reply src/include/target/Vantac_2400_RX.h]: Removed the comment

**Comment by CapnBry** [comment]
# TX target release checklist

## Module update
- [x] uart upload works
- [x] wifi upload works
- [ ] ~~passthrough upload works~~
 
## Display + control
- [x] RGB LED has correct color
- [ ] ~~logo shows up on startup~~
- [ ] ~~joystick control works in all directions~~
	
## LUA setup
- [x] uart connection works -> 5.25M
	  - open elrs lua script
	  - verify UART connection works
	  - test faster baud rates + retest connection
- [ ] ~~bind to rx via lua script works~~
- [x] check enable wifi works

## RX compability
- [x] module establishes link to rx of the same vendor (specify rx) -> Vantac RX
- [x] tlm data is received
- [x] module establishes link to rx of other vendor (specify other rx) -> EP2
  
## TX power
- [x] set tx power + verfiy correct output power 0> see above
- [x] set highest tx power and check for temperature issues after 10min
- [x] verify rssi dbm value at 1m
 
# RX target release checklist

## Module update
- [x] wifi upload works
    - [x] Range 10cm
    - [x] Range 1m
    - [x] Range 10m
- [x] passthrough upload works -> could not get to work, fixed
- [x] UART upload works
 
## TX power
- [x] verfiy tlm output power at 1m and PA working if included
 
## TX compability
- [x] module establishes link to tx of the same vendor (specify tx) -> Vantac Lite
- [x] module establishes link to tx of other vendor (specify other tx) -> FM30

## FC compability
- [x] connect to FC (specify FC) + verfiy that rx boots and FC gets channel values -> Omnibus F4

**Review by maybenikhil — DISMISSED** [review]

**Comment by pkendall64** [comment]
Finally go to test this and it's all good as @CapnBry says 👍

**Review by AlessandroAU — APPROVED** [review]

**Pull request merged**